wikitextparser/_template.py

```diff
@@ -123,7 +123,7 @@ def rm_first_of_dup_args(self) -> None:
         for a in reversed(self.arguments):
             name = a.name.strip(WS)
             if name in names:
-                del a[:len(a.string)]
```

Comment:
Nice catches! I'll definitely accept a pull request to change these to `del a[:]` form.